platform/darwin/include/mbgl/interface/native_apple_interface.h

```diff
@@ -10,7 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @required
 
-- (NSURLSessionConfiguration *)sessionConfiguration;
```

Comment:
any concern about removing exposure for the sessionConfig?

Reply:
Some - AFAICT the header file is exposed by CMake, but as `PRIVATE`. This is unlike `http_file_source.h` which is exposed via the `CMakeLists.txt` file. 

However, my question is: should this be exposed like `http_file_source.h`, since we're essentially using it like a public API?

cc @tmpsantos - what's the correct thing to do here? Do you know if we have other cases like this?

Reply:
Looks like it is already a public header for the darwin platform, because it is inside `platform/darwin/include/` instead of `platform/darwin/src/`.

Reply:
Removed the removal